docs/adr/0011-create-new-react-app-for-settings-redesign.md

```diff
@@ -0,0 +1,76 @@
+# Create a New React Application for the Settings Redesign Project
+
+- Deciders: Lauren Zugai, Les Orchard, Danny Coates, Wil Clouser
+- Date: 2020-03-19
+
+## Context and Problem Statement
+
+"Settings" refers widely to the `accounts.firefox.com/settings` page where users can manage their Firefox account. The "Settings Redesign Project" refers to a 2019/2020 project centered around giving this page a fresh user interface. The changes are fairly significant and will require a lot of component shuffling and some functionality refactoring. [See the PRD](https://docs.google.com/document/d/18zu7JCYIsUp8tUMJqb2uErNlzL9f6CQvefLy9HFZ4UY/edit?pli=1#heading=h.cf57dt1i8634).
+
+The FxA Engineering team desires to migrate from [Backbone and Mustache to React](https://github.com/mozilla/fxa/blob/master/docs/adr/0010-transition-fxa-from-backbone-to-react.md). This ADR addresses questions around what level of action we should take regarding this desire and the Settings Redesign Project - should a conversion to React be done at a later time, or are the changes significant enough to justify propelling us into React now?
+
+## Considered Options
+
+- Option A - Use the existing JS framework (Backbone) and templating library (Mustache), and make all changes in these files
+- Option B - Take the "100/0" approach by creating all new components with React, but modify existing Backbone/Mustache files
+- Option C - Take the "ground up" approach and create a new settings React application
+
+## Decision Outcome
+
+Chosen option: "Option C - Take the 'ground up' approach and create a new settings React application", because
+
+- The overall goal of the Settings Redesign Project is to offer more flexibility to expand Settings functionality. WeNew features have already been planned. It could be argued that heavy refactoring later (which would be needed for option A or B) would take just as much time as recreating this page with React now.
```

Comment:
~We~New ?

Reply:
Thanks @chenba for catching all my typos 😂